packages/replay/src/bin.ts

```diff
@@ -197,20 +209,35 @@ async function commandUploadRecording(id: string, opts: CommandLineOptions) {
 
 async function commandLaunchBrowser(
   url: string | undefined,
-  opts: Pick<CommandLineOptions, "warn" | "directory"> & {
-    browser: string | undefined;
-    attach: boolean | undefined;
-  }
+  opts: Pick<CommandLineOptions, "warn" | "directory"> & LaunchOptions
 ) {
   try {
     debug("Options", opts);
 
     const browser = fuzzyBrowserName(opts.browser) || "chromium";
     assertValidBrowserName(browser);
 
-    const attach = opts.attach || false;
+    await launchBrowser(browser, [url || "about:blank"], false, { ...opts, verbose: true });
```

Comment:
Correct me if I'm wrong but this is a breaking change. Do you know if users reply on this command and  if we need to have any migrate path?